app.js

```diff
@@ -0,0 +1,42 @@
+let books = [];
+const localData = JSON.parse(localStorage.getItem('Books'));
+const BookListContainer = document.querySelector('.books-container');
+
+if (localData != null) {
+  localData.forEach((item) => {
+    BookListContainer.innerHTML += `
+  <div>
+  <div>${item.name}</div>
+  <div>${item.author}</div>
+  <button class='btn-remove' id='${item.id}' onclick='remove(${item.id})'>Remove</button>
+  </div>
+  <hr>
+  `;
+    const book = {
+      id: item.id,
+      name: item.name,
+      author: item.author,
+    };
+    books.push(book);
+  });
+}
+
+const btnSubmit = document.querySelector('.add-book');
+const BookName = document.querySelector('.name');
+const BookAuthor = document.querySelector('.author');
+btnSubmit.addEventListener('click', () => {
+  const book = {
+    id: books.length,
+    name: BookName.value,
+    author: BookAuthor.value,
+  };
+  books.push(book);
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects.md#:~:text=Create%20a%20function%20to%20add%20a%20new%20book%20to%20the%20collection%2C%20with%20title%20and%20author), Please create a separate function to add a book to a book list and use it where ever you want.